collections/build-collections.py

```diff
@@ -18,7 +18,7 @@ def write_file(dict, path):
 
         s3 = boto3.resource('s3')
         print("writing:" + path)
-        s3.Object(manifest_bucket, path).put(Body=json.dumps(dict), ACL='public-read', ContentType='text/json')
+        #s3.Object(manifest_bucket, path).put(Body=json.dumps(dict), ACL='public-read', ContentType='text/json')
```

Comment:
Why did you comment out this code? Can you instead remove it (with a commit message explaining why).